dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-post-amendments.controller.js

```diff
@@ -28,8 +29,8 @@ exports.postTfmAmendment = async (req, res) => {
           status: CONSTANTS.AMENDMENT.AMENDMENT_STATUS.NOT_STARTED,
           version: 1
         };
-        if (latestCompletedAmendment) {
-          amendment.version = latestCompletedAmendment.version + 1;
+        if (latestCompletedAmendmentVersion) {
+          amendment.version = latestCompletedAmendmentVersion + 1;
```

Comment:
Do we need to typecast it to `Number()`?

Reply:
That won't be necessary as the version is always a number